docs/how-to/kafka-ui.md

```diff
@@ -0,0 +1,144 @@
+
+To activate the charmed Kafka UI application, integrate it with the charmed Apache Kafka application:
+
+```bash
+juju integrate kafka-ui kafka
+```
+
+After a few minutes, the charmed Kafka UI application should be in `active|idle` state.
+
+## Configure authentication
+
+Authentication is enabled by default on the charmed Kafka UI application for the internal `admin` user. To change the `admin` user's password, you need to define a Juju secret with your desired credentials and configure the charmed Kafka UI application to use that secret. The process is as follows:
+
+First, add a custom secret for the internal admin user with your desired password:
+
+```bash
+juju add-secret ui-secret admin='My$trongP4ss'
+```
+
+You will receive a secret ID in response, for example:
+
+```text
+secret:d4aph58sv8l31ign9590
+```
+
+Then, grant access to the secret with:
+
+```bash
+juju grant-secret ui-secret kafka-ui
+```
+
+Finally, configure the UI application to use the provided secret:
+
+```bash
+juju config kafka-ui system-users=secret:d4aph58sv8l31ign9590
+```
+
+## Access the Kafka UI
+
+To access the UI, simply open a web browser and point it to the `https://{KAFKA_UI_IP}:8080`. `KAFKA_UI_IP` is the IP address of the Kafka UI application, which you can either copy from the output of the `juju status` command, or retrieve it using the following command:
+
+```bash
+KAFKA_UI_IP=$(juju status --format json | jq -r '.applications."kafka-ui".units.[]."public-address"')
+```
+
+```{note}
+By default, charmed Kafka UI uses a self-signed certificate to secure communications. You need to instruct your web browser to trust this certificate. More details on how to do that for Firefox and Google Chrome browsers can be found below:
+
+- **Firefox** - [Set up Certificate Authorities (CAs) in Firefox](https://support.mozilla.org/en-US/kb/setting-certificate-authorities-firefox)
+- **Google Chrome** - [Set up TLS (or SSL) inspection on Chrome devices](https://support.google.com/chrome/a/answer/3505249?hl=en)
+```
+
+
+You should see an authentication page like below, in which you can use the `admin` username and the password configured before to log in.
+
+```
+todo: add pic
+```
+
+Once logged in, you can use the left menu to access the brokers, KRaft controllers, topics, schemas, and connectors configuration along with various monitoring metrics. To familiarise yourself with Kafbat's Kafka UI features, it is advised to consult the product's [official documentation](https://ui.docs.kafbat.io/).
+
+## Integrate charmed Kafka UI with other products
+
+The charmed Kafka UI operator can integrate with other charmed operators, including the charmed Kafka Connect and the charmed Karapace operators. For more information on these products and their use-cases, please refer to the [How to use Kafka Connect for ETL workloads](how-to-use-kafka-connect-for-etl-workloads) and [Schemas and serialisation](how-to-schemas-serialisation) guides.
```

Comment:
Suggestion: It's not mandatory, but will be easier to maintain if we split longer lines. It will limit the scope of future potential changes.

```suggestion
The charmed Kafka UI operator can integrate with other charmed operators, including the charmed Kafka Connect and the charmed Karapace operators.
For more information on these products and their use-cases, please refer to the
[How to use Kafka Connect for ETL workloads](how-to-use-kafka-connect-for-etl-workloads) and
[Schemas and serialisation](how-to-schemas-serialisation) guides.
```